pythagorean_thm.py

```diff
@@ -0,0 +1,27 @@
+'''
+This script will help solve the pythagorean theorem.
+'''
+
+import math
+
+def main(a, b, c):
```

Comment:
What are a, b, and c? Not good variable names.

Reply:
and what types of variables should they be ?